js/uws/index.js

```diff
@@ -2,13 +2,26 @@
 
 var WebSocketServer = require('uws').Server;
 var wss             = new WebSocketServer({ port: 3334 });
+var cluster         = require('cluster');
+
+if (cluster.isWorker) {
+  process.on('message', function(msg) {
+    wss.broadcast(msg);
+  });
+}
 
 function echo(ws, payload) {
   ws.send(JSON.stringify({type: "echo", payload: payload}));
 }
 
 function broadcast(ws, payload) {
-  wss.broadcast(JSON.stringify({type: "broadcast", payload: payload}));
+  var msg = JSON.stringify({type: "broadcast", payload: payload});
+  wss.broadcast(msg);
```

Comment:
Yeah, thought about it.  Then you need an `if` block that looks like this:

```javascript
if (cluster.isMaster) {
  wss.broadcast(msg);
} else {
  process.send(msg);
}
```

So that `uws/index.js` can still be executed as a single-process server.  It's either way really.  I thought if @jackc ran the benchmarks again it would be useful/enlightening to those not too familiar with node to see the difference between clustered/non-clustered usage.